src/gConsent/manage/approveAccessRequest.test.ts

```diff
@@ -145,6 +146,9 @@ describe("approveAccessRequest", () => {
     accessGrantVc = await mockAccessGrantVc();
     consentRequestVc = await mockConsentRequestVc();
     consentGrantVc = await mockConsentGrantVc();
+    cacheProvider(new URL(MOCKED_ACCESS_ISSUER).href, {
```

Comment:
There is a bunch of changes like this in this PR, it pre-fills the cache so that the test doesn't go to fetch the mock VC provider config.